pynta/main.py

```diff
@@ -128,6 +128,8 @@ def generate_slab(self,skip_launch=False):
         #construct slab with optimial lattice constant
         slab = slab_type(self.metal,self.repeats,a,self.vacuum)
         slab.pbc = (True, True, False)
+        if self.software.lower() == 'vasp' or self.software.lower() == 'dp':
```

Comment:
Does this mean that in VASP we have a slab above the vacuum as well as below? Are we sure this doesn't have other consequences for the workflow?

Reply:
So far from my testing, this does not appear to impact the workflow. 
I don't think it would make a difference since the periodic slabs along the z-axis are so far away. 
Maybe we could meet and discuss. I'd be interested in hearing your thoughts.